src/main/java/net/atos/zac/zoeken/converter/AbstractZoekObjectConverter.kt

```diff
@@ -0,0 +1,10 @@
+package net.atos.zac.zoeken.converter
+
+import net.atos.zac.zoeken.model.ZoekObject
+import net.atos.zac.zoeken.model.index.ZoekObjectType
+
+abstract class AbstractZoekObjectConverter<ZOEKOBJECT : ZoekObject> {
```

Comment:
```suggestion
abstract class AbstractZoekObjectConverter<out ZOEKOBJECT : ZoekObject> {
```

Reply:
nice! fixed this and removed 'out' keyword from code calling these converters.